sonic_platform_base/sonic_xcvr/api/public/cmis.py

```diff
@@ -43,6 +43,18 @@ def get_model(self):
         '''
         return self.xcvr_eeprom.read(consts.VENDOR_PART_NO_FIELD)
 
+    def get_cable_type(self):
+        '''
+        This function returns the cable type of the module
+        '''
+        return "Length Cable Assembly(m)"
```

Comment:
a static string?

Reply:
The cable_type was already hardcoded for CMIS. Previous code
```
 "cable_type": "Length Cable Assembly(m)",
```
However, since I am now calling the function get_cable_type from xcvrd for finding NPU SI settings, I am returning the hardcoded string to adhere to the previous behavior.

Reply:
![image](https://github.com/sonic-net/sonic-platform-common/assets/19700240/c4df5f75-2091-4901-943d-9d93f9f8fba8)

if I understand correctly "Length Cable Assembly(m)" should be referring to the above field of CMIS, not sure how can it be related to "cable_type"?

Reply:
I have now renamed this API to get_cable_length_type.
We have the API get_cable_length for finding the cable length.